__tests__/app.test.js

```diff
@@ -295,38 +287,76 @@ test("400 - handles bad path - no such key", () => {
+    return request(app)
       .get("/api/articles")
       .expect(200)
       .then(({ body }) => {
         const { articles } = body;
         expect(articles).toBeInstanceOf(Array);
-        expect(articles.length).toBe(5);
-            articles.forEach((article) => {
-              expect(article).toEqual(
-                expect.objectContaining({
-                  author: expect.any(String),
-                  title: expect.any(String),
-                  article_id: expect.any(Number),
-                  body: expect.any(String),
-                  topic: expect.any(String),
-                  created_at: expect.any(String),
-                  votes: expect.any(Number),
-                  comment_count:expect.any(String)
-                })
-              );
-         });
-    });
-  
+        expect(articles.length).toBe(12);
+        articles.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+        expect(articles).toBeSortedBy("created_at", { descending: true });
+      });
   });
-})
+});
 
+describe("9. GET /api/articles/:article_id/comments ", () => {
+  test("200 - different article updates article and returns updated article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);
+        expect(comments.length).toBe(11);
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              body: expect.any(String),
+              votes: expect.any(Number),
+              author: expect.any(String),
+              article_id: expect.any(Number),
+              created_at: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+
+  test("404 - handles bad path - no comments or no article", () => {
+    return request(app)
+      .get("/api/articles/99999/comments")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("article 99999 - has no comments");
+      });
+  });
 
+  test("400 - handles bad path - string for article_id", () => {
+    return request(app)
+      .get("/api/articles/invalid_path/comments")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("article id must be a number");
+      });
+  });
+});
```

Comment:
Would like a test for if the article DOES exist, but has NO comments. We should return an empty array in this case